app/src/main/java/com/nervousfish/nervousfish/activities/CreateProfileHelper.java

```diff
@@ -79,6 +83,22 @@
 
     /**
      * @param input The {@link EditText} to evaluate.
+     * @return The resultcode of validateIban.
+     */
+    Constants.InputFieldResultCodes validateIban(final EditText input) {
+        final String iban = input.getText().toString();
+        if (EMPTY_STRING.equals(iban) || IBANVerifier.isValidIBAN(iban)) {
```

Comment:
Doesn't `IBANVerifier.isValidIBAN(iban)` return false if `iban` is an empty string?